test/unit/com/unifina/domain/CustomStreamIDValidatorSpec.groovy

```diff
@@ -31,6 +31,7 @@ class CustomStreamIDValidatorSpec extends Specification {
 		"sub.my-domain.eth/foo//bar" | false
 		"foobar.eth/abc/def" | false
 		"sub.my-domain.eth" | false
+		"0xAbcdeabCDE123456789012345678901234567890" | false
```

Comment:
@teogeb How about adding an e2e test for this also?

Reply:
Most of the Groovy specs get throw to the trash bin when work starts on BACK-163. Probably not the case in this test case 😀